<launch>
    <arg name="marker_frame"    default="aruco_marker_frame"/>
    <arg name="ref_frame"       default=""/>  <!-- leave empty and the pose will be published wrt param parent_name -->
    <arg name="corner_refinement" default="LINES" /> <!-- NONE, HARRIS, LINES, SUBPIX -->
    <arg name="node_namespace" default="bundle" />
    <!-- <rosparam command="load" file="$(find aruco_ros)/cfg/config.yaml" ns="$(arg node_namespace)" /> -->
    <node pkg="aruco_ros" type="bundle" name="aruco_bundle" output="screen" >
        <remap from="camera_info" to="/camera/color/camera_info" />
        <remap from="image" to="/camera/color/image_raw" />
        <param name="image_is_rectified" value="True"/>
        <param name="reference_frame"    value="$(arg ref_frame)"/>   <!-- frame in which the marker pose will be refered -->
        <param name="camera_frame"       value="camera_color_aruco"/>
        <param name="marker_frame"       value="$(arg marker_frame)" />
        <param name="corner_refinement"  value="$(arg corner_refinement)" />
    </node>

</launch>

